app/models/alaveteli_pro/request_summary_category.rb

```diff
@@ -0,0 +1,40 @@
+class AlaveteliPro::RequestSummaryCategory < ActiveRecord::Base
+  has_and_belongs_to_many :request_summaries,
+                          :class_name => AlaveteliPro::RequestSummary
+
```

Comment:
I did think about that, but I wasn't sure whether it really gained us anything? We have the categories we rely on in the seeds file already, and I can't think of a good reason to lock down the creation of any more, except perhaps for having another way of making it explicit which ones are available. If that sounds like a good idea, I can add it, what do you reckon?